inject-dist/observers.js

```diff
@@ -172,6 +172,9 @@ function initializeObserver(pathname) {
 
         // Return the original feature.
         return original;
+      },
+      set: () => {
+        return original;
```

Comment:
If an application attempts to overwrite one of these properties, would this prevent it from working, since it's not actually setting the new value?

Worth testing if this works:

```suggestion
      set: (new) => {
        original = new;
```

Reply:
The above gives a number of issues (`undefined` or `null`), while my proposed solution removes any setters which are added through the app's code. So neither is a good solution as yet. I will try investigate the `null`/`undefined` to understand what's going on .

In the meantime I tried to rewrite it using a `proxy` but we are unable to proxy all objects. For example, it is not allowing me to proxy `window.navigator`.